File: city-list-back/src/main/java/com/helmes/makedon/citylist/domain/City.java
package com.helmes.makedon.citylist.domain;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;

/**
 * @author Yahor Makedon
 */
@Entity
@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class City implements BaseBean {
	private static final long serialVersionUID = 1L;

	@Id
	@GeneratedValue(strategy = GenerationType.SEQUENCE)
	private Long id;
	@Column(nullable = false)
	private String name;
	@Column(nullable = false)
	private String photoUrl;
}

File: city-list-front/src/app/domain/city.ts
export class City {
  private _id: number | undefined;
  private _name: string | undefined;
  private _photoUrl: string | undefined;

  constructor() {
  }

  get id(): number | undefined {
    return this._id;
  }

  set id(value: number | undefined) {
    this._id = value;
  }

  get name(): string | undefined {
    return this._name;
  }

  set name(value: string | undefined) {
    this._name = value;
  }

  get photoUrl(): string | undefined {
    return this._photoUrl;
  }

  set photoUrl(value: string | undefined) {
    this._photoUrl = value;
  }
}
